tests/test_users/conftest.py

```diff
@@ -47,36 +47,38 @@
         "password_repeat": "janusz1488"
         }
 
-user_update_data = {
-        "first_name": list_of_user_register_schemas[0].first_name,
-        "last_name": list_of_user_register_schemas[0].last_name,
-        "email": "kowal@mailedit.com",
-        "birth_date": "1983-10-12",
-        "username": list_of_user_register_schemas[0].username
-        }
+EXISTING_USER_DATA = LIST_OF_USER_REGISTER_SCHEMAS[0]
+
+USER_UPDATE_DATA = {"email": "kowal@mailedit.com"}
+
+@pytest.fixture
+def register_existing_user_data() -> UserRegisterSchema:
+    return EXISTING_USER_DATA
+
     
-@pytest.fixture(scope="package", autouse=True)
+@pytest.fixture
 def db_users(sync_session: Session):
-    return [register_user(sync_session, user) for user in list_of_user_register_schemas]
+    delete_all_users(sync_session)
+    return [register_user(sync_session, user) for user in LIST_OF_USER_REGISTER_SCHEMAS]
```

Comment:
nie musisz userow usuwac, z tego co pamietam to conftest twoj juz cofa transakcje po test case jednym